package.json

```diff
@@ -21,6 +21,7 @@
     "nodemon": "^1.18.1"
   },
   "dependencies": {
+    "express": "^4.16.3",
```

Comment:
Remembering to install needed dependencies not already in the package.json file, nice 👍